### PR TITLE
fisrt commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
https://pavel-gutsal.github.io/react_decompose/

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+import './Header.css';
 
-// Create a Header function returning the HTML of header block
+const Header = () => (
+  <header className="header">
+    <h1 className="header__title">Site Name</h1>
+    <nav className="navigation">
+      <a className="navigation__link" href="#about">About</a>
+      <a className="navigation__link" href="#services">Services</a>
+      <a className="navigation__link" href="#contact">Contact</a>
```
Comment (valerii-derkach) [inline src/components/Header/Header.jsx:10]: navigation is separate component, so create a file and move it there
render navigation inside header

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -1 +1,42 @@
...
+  opacity: 0.7;
+  padding: 0 0.5em;
+  color: black;
+  transition: opacity 0.2s ease-in-out;
+}
+
+.navigation__link:hover,
+.navigation__link:focus {
+  opacity: 1;
+}
```
Comment (valerii-derkach) [inline src/components/Header/Header.css:34]: move navigation styles to separate component

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

**Comment by pavel-gutsal** [comment]
Split the `App.js` into smaller components basing on CSS blocks. CSS should be split too.
- Create a sub folder in `components` per component and put `js` and `css` files there.
- There should be 3 new components: `Article`, `Header`, `Welcome`.

So I added navigation as separate component even if its not in the task , on request of the mentor , please don`t ask me to remove it)))

**Review by valerii-derkach — APPROVED** [review]
Everything good! Well done!

**Pull request closed without merging**